code/modules/mob/holder.dm

```diff
@@ -63,7 +63,7 @@
 	name = "maintenance drone"
 	desc = "It's a small maintenance robot."
 	icon_state = "drone"
-
+	
 /obj/item/holder/Runtime
```

Comment:
The only other thing I'd say on this is you could change the pathing so they all inherit and don't have to be 100% written out again, but that's not strictly necessary. Just removes having to specify icon_state for each one over and over.
Would probably look something like this
`/obj/item/holder/cat
/obj/item/holder/cat/kitten
/obj/item/holder/cat/jones
/obj/item/holder/cat/black
/obj/item/holder/cat/black/runtime`

Reply:
Haven't forgotten, but I'll look into this in a few. It can probably be simplified yeah.